appendix.Rmd

```diff
@@ -49,11 +51,11 @@ pkgs = gsub("\\(.*)", "", pkgs) # Remove versions from packages
 ext_pkgs = c("rio", "feather")  # issues installing on travis
 to_install = !pkgs %in% rownames(installed.packages())
 
-if(sum(to_install) > 0){
+if(sum(to_install) > 0 & !is_travis){
   install.packages(pkgs[to_install])
 }
 
-if(!all(ext_pkgs %in% rownames(installed.packages()))) {
+if(!all(ext_pkgs %in% rownames(installed.packages())) & !is_travis) {
```

Comment:
This piece of code is only executed on now travis machines: `eval=(!is_travis)` So the `& !is_travis` isn't needed.